### ISSUE TITLE
included file inside table cell doesn't render footnotes

### ISSUE THREAD

**Issue by philip-stuckey** [asciidoctor/asciidoctor-pdf#2410]
hello, I have a fairly complicated document, and I tried putting two subsections in a two column format by putting them in a table, but I found the footnotes in the subsections disappeared. I have narrowed it down to this minimum-working example.
Everything in the block comment is supposed to be a separate file that the first one is including 

````asciidoctor
[cols="1a"]
|===
|include::separate-document.adoc[]
|this seems to work fine{empty}footnote:[this footnote shows up]
|===
````

in separate-document.adoc
````
when this file is included, the footnote is nowhere to be seen
footnote:[this footnote should appear in the including document, so here it should show up twice]
````

here are the results
![image](https://user-images.githubusercontent.com/7987932/227644402-740cf056-8cc0-42bb-946f-cc22c7bedac9.png)

see how the first footnote at the bottom is "[2]"?
when I ran this in normal asciidoctor the footnotes appear in the bottom of their respective cells as opposed to the bottom of the page. 
note that this is in asciidoctor-pdf version 2.3.4 using asciidoctor 2.0.18

**Comment by philip-stuckey** [comment]
oops, this is a dup, sorry

**Comment by philip-stuckey** [comment]
seems related to https://github.com/asciidoctor/asciidoctor-pdf/issues/1777 except the footnote isn't rendered only if it's included from another document

**Comment by mojavelinux** [comment]
The include directive is a red herring here. This happens with or without the include. For some reason, the first footnote is being dropped. I'm investigating.

**Comment by mojavelinux** [comment]
Apply this fix to main, then backport to the 2.3.x branch.
